src/generate-sourcelink-json.csharp

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env /Library/Frameworks/Mono.framework/Commands/csharp
+
+using System.IO;
+using System.Text;
+
+var args = Environment.GetCommandLineArgs ();
+
+var latestCommit = args [2];
+var src = args [3];
+var outputPath = args [4];
```

Comment:
Adjust to use the Args variable: https://github.com/xamarin/xamarin-macios/commit/744902f014a3cb38c6e0a9b49e330a864f7403c5